### DISCUSSION TITLE
[BUG] Not able to connect to Sharepoint Online using stored credentials

### DISCUSSION THREAD

**Discussion by jessenoyes-cire** [pnp/powershell#3279] [Q&A]
When trying to connect to Sharepoint online using PnP stored credentials in Windows Powershell, I'm getting thrown a error message that states:
`Connect-PnPOnline: There was an error parsing the WS-Trust response from the endpoint.
This may occur if there are issues with your ADFS configuration. See https://aka.ms/msal-net-iwa-troubleshooting for more details.
Enable logging to see more details. See https://aka.ms/msal-net-logging. Error Message: Request to the endpoint timed out.`

This was working as of last week and using the `-Interactive` switch to sign in manually works using both my own credentials and that which are being stored in the credential manager.

**Comment by myr-mattson** [comment]
Same issue here.
